### PR TITLE
release/5.5 -> develop

### PR DESCRIPTION
**Pull request by AliSoftware**
Code freeze for 5.5

NB: This PR already includes the Release Notes validated by Editorial [p2JRYi-57q-p2]

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,7 +1,11 @@
+*** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
```
Comment (jkmassel) [inline RELEASE-NOTES.txt:1]: Will this change break the "automatically add the version number to the top of the file" functionality?

**Review by jkmassel — APPROVED** [review]
LGTM – just one question around the release notes file change.

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (AliSoftware) [reply RELEASE-NOTES.txt:1]: Yes, it actually already does (as you can see from commit history 😓 )

This new note has been added like just 2 releases ago, and we haven't got around updating our scripts to take it into consideration just yet, so for now I update its position manually. Hopefully we'll get around updating the script to something like "ignore leading lines that start with `*** ` when adding the new version number" or something like that (so we can add even more instructions/notes here in the future if we need to)

**Comment by AliSoftware** [comment]
Closing in favor of https://github.com/woocommerce/woocommerce-ios/pull/3162 to fix conflict resolution

**Pull request closed without merging**